app/views/steps/income/dependants/edit.html.erb

```diff
@@ -17,7 +17,7 @@
       <%= f.fields_for :dependants do |c| %>
         <% index = c.index + 1 %>
 
-        <%= c.govuk_fieldset legend: { text: t('.dependant_legend', ordinal: index.ordinalize_fully.capitalize) }, id: "dependant_#{index}", class: 'govuk-!-margin-top-4'  do %>
+        <%= c.govuk_fieldset legend: nil, id: "dependant_#{index}", class: 'govuk-!-margin-top-4'  do %>
```

Comment:
I think this is an accessibility issue to set the legend to nil, as it means there's no label for the fieldset for screenreaders etc. If you want to just not display the legend you can add `hidden: true` as an attribute to the legend?

Reply:
Ooh good to know, i'll make that change